app/models/user.rb

```diff
@@ -1,5 +1,10 @@
 class User < ApplicationRecord
   has_many :posts
+  has_many :followships
+  has_many :followees, through: :followships
+
+  has_many :inverse_followships,class_name: "Followship",foreign_key:"followee_id"
+  has_many :inverse_followees, through: :inverse_followships,source: :user
   # Include default devise modules. Others available are:
```

Comment:
please use meaningful name.

Reply:
I think these names are quite meaningful.

Reply:
How can use of word inverse is meaningful. Your code to logical only for you. You are not writing code just for yourself but for other developers also. Will you always be there to explain them.